docs/docs/credits.md

```diff
@@ -25,4 +25,4 @@ Relevant repositories include [bayesiains/nsf](https://github.com/bayesiains/nsf
 - **On Contrastive Learning for Likelihood-free Inference**<br>Durkan C.,
   Murray I., and Papamakarios G.(ICML 2020) <br>[[PDF]](https://arxiv.org/abs/2002.03712).
```

Comment:
There are SRE, SNL missing here.

Reply:
Not solving in this issue, noted in #203